docs/resources/modelarts_workspace.md

```diff
@@ -0,0 +1,96 @@
+---
+subcategory: "AI Development Platform (ModelArts)"
+---
+
+# huaweicloud_modelarts_workspace
+
+Manages a Modelarts workspace resource within HuaweiCloud.  
+
+## Example Usage
+
+### Create a public workspace
+
+```hcl
+resource "huaweicloud_modelarts_workspace" "test" {
+  name        = "demo"
+  description = "This is a demo"
+}
+```
+
+### Create an internal workspace
+
+```hcl
+variable "user_id" {}
+resource "huaweicloud_modelarts_workspace" "test" {
+  name        = "demo"
+  description = "This is a demo"
+  auth_type   = "internal"
+  grants {
+    user_id = var.user_id
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `name` - (Required, String) Workspace name, which consists of 4 to 64 characters.  
+  Only chinese and english letters, digits, hyphens (-), and underscores (_) are allowed.
+  **default** is the name of the default workspace reserved by the system.
+
+* `description` - (Optional, String) The description of the worksapce.  
+
+* `enterprise_project_id` - (Optional, String, ForceNew) The enterprise project id of the worksapce.  
+  Value 0 indicates the default enterprise project.
+
+  Changing this parameter will create a new resource.
+
+* `auth_type` - (Optional, String) Authorization type.  
+  Value options are as follows:
+    + **public**: public access within the tenant.
+    + **private**: Only the creator and master account can access.
+    + **internal**: Creator, master account, and specified IAM sub-accounts can access.
```

Comment:
In console, it's called **main account**

Reply:
fix